proposals/0390-noncopyable-structs-and-enums.md

```diff
@@ -573,6 +573,14 @@ guard let condition = getCondition() else {
 use(x)
 ```
 
+For the purposes of the following discussion, a closure is considered nonescaping
+in the following cases:
+
+- if the closure literal appears as an argument to a function parameter of
+  non-`@escaping` function type, or
+- if the closure literal is assigned to a local `let` variable, that does not
```

Comment:
Is it worth mentioning here that we are following the behavior shown by how the compiler already handles inout/escaping closures?